packages/rn-tester/Podfile.lock

```diff
@@ -558,36 +558,36 @@ SPEC CHECKSUMS:
-  React-jsinspector: 8db1ac014c0d38a6673b0fdb83642d85fd4cb579
-  React-logger: f2c1ddea67269cb504924a5c07ae508a6409b987
-  React-perflogger: caaa9c52857ae620ef60bcf7f9b6411a16948872
-  React-RCTActionSheet: 8cf9c718ee9088bd380a1107cdcb7e638b7fc817
-  React-RCTAnimation: e799c0f4d7185323b5c8501e54ae004df40000f9
-  React-RCTBlob: 328da29bdc57bf6d374a5d4718a7da9220c104c1
-  React-RCTImage: 56acd81b3c1408eb8314373feb6ca29b274ede71
-  React-RCTLinking: e4f1793bf7a767d7c6fa78fdeb68b4cb7ad42619
-  React-RCTNetwork: 0c12d55775374c0bfa3d165f584b615290de0e14
-  React-RCTPushNotification: 29f85b459afbdf42403acb17626f3b143193146f
-  React-RCTSettings: c70de3a7ee87019e7f0cbc10e8abc0e6a77a2dbb
-  React-RCTTest: ece529f0bc614cc3940354d524ebe06d24ed2115
-  React-RCTText: e0423ccc4b50d83d54cf6a922e57a6b0113bb297
-  React-RCTVibration: 1de2bc1d267f5eef12bba7e2fab65c07d712f849
-  React-runtimeexecutor: 43e26ebbe2bddb0a6967638887c2f7e08869f06e
+  RCTRequired: f12ad50a43367f98710e6362c1bdbeef0ea5245a
+  RCTTypeSafety: bc2d2db0e24cdcd3f5494fe5634024c0f20c0694
+  React: f6928da5ac8761ea569ac2b05de808c1d7b075a7
+  React-callinvoker: 4f06162a050a3d4543bced3b8f03a1751cd66925
+  React-Core: 79783d44c69157490c468a28f6fc01ebe0d23e43
+  React-CoreModules: 4def0259d36fb9db39f7936f2dfea0b5aff0d68a
+  React-cxxreact: 3662230abd4025885f2d77e7f0909bc91edb8f95
+  React-jsi: 2e4e2870285e26a23b7f2f950e8b73d1de722427
+  React-jsiexecutor: 1754d078d34254aeb3881ab2b966b7f13638584d
+  React-jsinspector: d6f952abeefc9dbdabc0932bf774f5f7a462a31f
+  React-logger: 8b67df82c8ca64f4f13f12344b2af4259c2d4611
+  React-perflogger: 033848a56d5ee42a5ef3b8989958101e2cccf0dd
+  React-RCTActionSheet: 87b9e8cea6e12951a2bc1ddfe9c491797d4e9bfa
+  React-RCTAnimation: b409786e0865b62fc7f8700d9e378e917507906a
+  React-RCTBlob: 0e78d703e6fc2b2d31ec4595aab838b30771b497
+  React-RCTImage: 577bff41682f5c75c74f154bf1b63bec11a9fd45
+  React-RCTLinking: 08547c7f9bdad04991215db911c195448645a92a
+  React-RCTNetwork: eddcc4c69ff76f172de31369f9a571683330e387
+  React-RCTPushNotification: 1aeddd1549c5e8ea77ddc468b961fa38de5482ac
+  React-RCTSettings: b016199203cf366f7a1336e8896a22acd7024887
+  React-RCTTest: 13e9f35afc4bcf2a724c3b47e3531cd5bf54c1a1
+  React-RCTText: f2960f96522f049f7b727e646951434ac490b3cd
+  React-RCTVibration: f2450721ed5fdc45bf087e457fc35e8828d99e2b
+  React-runtimeexecutor: a839145ce9d28fb3d152a3aa1584994560159468
   React-TurboModuleCxx-RNW: f2e32cbfced49190a61d66c993a8975de79a158a
-  React-TurboModuleCxx-WinRTPort: 41bff88d280cb2270c4980c8de1fb44f0bba908d
-  ReactCommon: b390476fd1380f1cd943187374ec2cacb834c3de
-  Yoga: ba56cce168e5cce27fc1c888d618bc1cac1ea1df
+  React-TurboModuleCxx-WinRTPort: 413735d46b568d10edfd1f05643d9e777f2a013b
+  ReactCommon: 2cf97e7f7e9ed1d9a14e2dbbd9c671e5074d32be
+  Yoga: 8774604124eb5f6eacf994b14d41cd6e4e383cbc
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
 PODFILE CHECKSUM: c7a7110b242497f2bf323ba74caedb9ee61ee05e
 
-COCOAPODS: 1.11.3
```

Comment:
Curious where this diff came from. Did we actually run pod install with this merge? Or perhaps we locally ran it in the main branch of our repo with 1.11.3 while someone at Meta ran it with 1.11.2.

I don't think this is an issue at all, Just thought it interesting.

Reply:
It might have something to do with the fact that I still have CocoaPods 1.11.2 on my main machine. Either way, I assume it should still be fine, especially since packages/rn-tester/Podfile.lock will get regenerated anyway whenever validating that RNTester works.